staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -468,6 +468,24 @@ func (v *VirtualMachineInstance) IsMigratable() bool {
 	return false
 }
 
+func (v *VirtualMachineInstance) IsMigrating() bool {
```

Comment:
The goal of removing utils files is a good one.
I do have a question about moving stuff to v1/types.. This is the first time I see that there are functions under types file. I dont think the other functions really fit in this file and so this one is even more complex and prone to changes which Im not sure this file is the right place for it..

Reply:
Hi @ShellyKa13, thank you for the review!
Would you suggest to move these functions to a dedicated file under the same package?

Reply:
@ShellyKa13 @fossedihelm Are we talking about a general approach of not adding functions to v1/types?
If so we need to send an email stating the new best practice. Until recently such PRs were accepted, recent one was [this](https://github.com/kubevirt/kubevirt/commit/d7a736606368efe600efc394d0537dadf3ca6e4a)

Reply:
About the best practice, we should first find an agreement

Reply:
@fossedihelm I think that if we will move other functions from this file we will break backward compatibility because this package is being released separately from the kubevirt module.

Reply:
@fossedihelm I am trying to find the common ground in order to find an agreement. On which of the following subjects do we have an agreement:
* Deprecating `pkg/util`
* Moving `IsMigrating` from `pkg/util/migration` in general.
* Moving 'IsMigrating` to be a method of the VMI struct.

Reply:
> @fossedihelm I am trying to find the common ground in order to find an agreement. On which of the following subjects do we have an agreement:
> 
> * Deprecating `pkg/util`

We should deprecate the package named `util`; the folder name could be ok, as long as it will not be used as a package.
> * Moving `IsMigrating` from `pkg/util/migration` in general.

Having said the above, that func could be moved to `/pkg/util/vmi/vmi.go` (many other functions can be moved here from `/pkg/util/util.go`)
> * Moving 'IsMigrating` to be a method of the VMI struct.

This cannot be achieved without adding it to `types` :) 

This is my POV, I would like to ear other feedback, since I might be an isolated case :slightly_smiling_face: